server/src/main/kotlin/com/cuhacking/atlas/server/Server.kt

```diff
@@ -21,18 +26,46 @@ class Server : CliktCommand() {
     @Suppress("MagicNumber")
     val port by option("--port", help = "Port for the server to listen on").int().default(8080)
 
+    private val test by option(
+        "--test",
+        "-t",
+        help = "Validate data against the schema without starting the server"
+    ).flag()
+
     override fun run() {
-        verifyData()
+        if (!verifyData()) {
+            throw RuntimeException()
+        }
 
-        embeddedServer(CIO, port, module = {
-            install(AutoHeadResponse)
-            dataModuleFactory(file)
-        }).start(wait = true)
+        if (!test) {
+            embeddedServer(CIO, port, module = {
+                install(AutoHeadResponse)
+                install(CallLogging)
+                dataModuleFactory(file)
+            }).start(wait = true)
+        }
     }
 
-    internal fun verifyData() {
-        // Will throw error if file is not a valid FeatureCollection
-        file.toFile().readText().toFeatureCollection()
+    private fun verifyData(): Boolean {
+        var valid = true
+
+        val stream = javaClass.classLoader.getResourceAsStream("data.schema.json")
+        val rawSchema = JSONObject(JSONTokener(stream))
+        val schema = SchemaLoader.load(rawSchema)
+
+        val json = file.readText()
+
+        try {
+            schema.validate(JSONObject(json))
+        } catch (e: ValidationException) {
+            valid = false
+            println(e.message)
+            e.causingExceptions.map(ValidationException::message).forEach(::println)
+        } catch (e: Exception) {
+            println(e)
```

Comment:
Are there any other particular errors we're expecting this to throw? Will returning true in any of those cases cause issues?